requirements.txt

```diff
@@ -4,112 +4,117 @@
 #
 #    pip-compile
 #
-asgiref==3.5.2
-    # via django
-attrs==22.2.0
-    # via pytest
-cachetools==5.2.0
+annotated-types==0.6.0
+    # via pydantic
+asgiref==3.8.1
+    # via
+    #   django
+    #   django-cors-headers
+cachetools==5.3.3
     # via django-helusers
-certifi==2023.7.22
+certifi==2024.2.2
     # via
     #   requests
     #   sentry-sdk
-cffi==1.15.1
+cffi==1.16.0
     # via cryptography
-charset-normalizer==2.1.1
+charset-normalizer==3.3.2
     # via requests
-coverage[toml]==7.2.1
+coverage[toml]==7.4.4
     # via pytest-cov
-cryptography==41.0.3
+cryptography==42.0.5
     # via -r requirements.in
 deprecation==2.1.0
     # via django-helusers
-django==3.2.21
+django==4.2.11
     # via
     #   -r requirements.in
     #   django-anymail
     #   django-cors-headers
     #   django-helusers
     #   django-mailer
     #   django-ninja
-django-anymail==10.0
+django-anymail==10.3
     # via -r requirements.in
-django-cors-headers==3.13.0
+django-cors-headers==4.3.1
     # via -r requirements.in
-django-environ==0.9.0
+django-environ==0.11.2
     # via -r requirements.in
-django-helusers==0.8.1
+django-helusers==0.11.0
     # via -r requirements.in
-django-mailer==2.2.1
+django-mailer==2.3.1
     # via -r requirements.in
-django-ninja==0.19.1
+django-ninja==1.1.0
     # via -r requirements.in
-ecdsa==0.18.0
+ecdsa==0.19.0
     # via python-jose
-exceptiongroup==1.1.1
+exceptiongroup==1.2.0
     # via pytest
 gunicorn==21.2.0
     # via -r requirements.in
-idna==3.4
+idna==3.7
     # via requests
 iniconfig==2.0.0
     # via pytest
 lockfile==0.12.2
     # via django-mailer
-packaging==21.3
+packaging==24.0
     # via
     #   deprecation
     #   gunicorn
     #   pytest
-pluggy==1.0.0
+pluggy==1.4.0
     # via pytest
-psycopg2-binary==2.9.6
+psycopg2-binary==2.9.9
```

Comment:
Doesn't need to be resolved in this PR, but it's recommended to use [source psycpg2 instead of the binary one](https://www.psycopg.org/docs/install.html#psycopg-vs-psycopg-binary).

Reply:
Changed.